main.py

```diff
@@ -1,20 +1,20 @@
+from logger import logger
```

Comment:
why re-order these imports? The system path appended should be ahead of local class import.